images/alpine/cloudbuild.yaml

```diff
@@ -1,19 +1,60 @@
 steps:
+# Step 1: Set up multiarch support.
   - name: gcr.io/cloud-builders/docker
     args:
+    - run
+    - --privileged
+    - multiarch/qemu-user-static
+    - --reset -p yes
+
+# Step 2: Create buildx builder.
+  - name: gcr.io/cloud-builders/docker
+    env:
+      - DOCKER_CLI_EXPERIMENTAL=enabled
+    args:
+    - buildx
+    - create
+    - --use
+    - --name=k8s-testimages
+
+# Step 3: Build multi-arch images, then push.
+  - name: gcr.io/cloud-builders/docker
+    env:
+      - DOCKER_CLI_EXPERIMENTAL=enabled
+    args:
+    - buildx
     - build
+    - --platform=linux/amd64,linux/arm64,linux/ppc64le
```

Comment:
Any reason for this particular list of platforms?

Reply:
Hi,
At present, kubevirt/istio is ready to do the prow ci for these 3 platforms: x86, powerpc, arm64.
And images/alpine is the basic image for other images, such as: git/sidecar/initupload...

Reply:
> Any reason for this particular list of platforms?

Hi, Ben,
We are trying to enable CI test for kubevirt/kubeflow/istio project on ARM. And facing a problem that in order to trigger a prow job on ARM cluster, prow utility images need to run on ARM cluster. So we are wandering if we can make at least these utility images (sidecar/initupload/entrypoint/clonerefs) support multi-arch. We have blocked here for really a long time.
Thanks a lot!

Reply:
@BenTheElder 
Any comments?